static/js/resize.js

```diff
@@ -56,6 +56,7 @@ function get_new_heights() {
         Number.parseInt($(".narrows_panel").css("marginTop"), 10) -
         Number.parseInt($(".narrows_panel").css("marginBottom"), 10) -
         $("#global_filters").safeOuterHeight(true) -
+        $("#private_messages_section").safeOuterHeight(true) -
```

Comment:
I think adding this is a bug -- it means that if the private message section is tall, it's not possible to scroll in the streams region. The right thing probably involves this new "PRIVATE MESSAGES" widget and "STREAMS" sharing a scrollable region, with sticky headers or something.

Reply:
I would expect fixing this to also mean we get to delete the calls to this module from the pm_list module.